Customizing/global/plugins/Services/Cron/CronHook/UserOrguImport/classes/Factory.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * General plugin factory. Also creates sub-namespace-factories.
+ */
+
+namespace CaT\IliasUserOrguImport;
+
+use CaT\UserOrguImport as UOI;
+
+class Factory
```

Comment:
I think this is not a `Factory` in reality, but a self-made DIC. I mean, this initializes services, not distinct objects, right? One `UserFactory` is not different from another, right?